pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -672,6 +711,34 @@ func (o *SyncOptions) enableSyncerForWorkspace(ctx context.Context, config *rest
 	return string(saTokenBytes), syncerID, syncTarget, nil
 }
 
+func mergeLatestResourceSchema(apiExport *apisv1alpha1.APIExport, resourceToSync []string) (*apisv1alpha1.APIExport, bool) {
+	desiredResourceGroup := sets.NewString()
+	var modified bool
+	for _, schema := range apiExport.Spec.LatestResourceSchemas {
+		comps := strings.SplitN(schema, ".", 3)
+		if len(comps) != 3 {
+			continue
+		}
+		desiredResourceGroup.Insert(fmt.Sprintf("%s.%s", comps[1], comps[2]))
+	}
+	for _, resource := range resourceToSync {
+		nameParts := strings.SplitN(resource, ".", 2)
+		resourceName := nameParts[0]
+		apiGroup := "core"
```

Comment:
Is this a constant we can import?

Reply:
Do you mean, should we use:
`corev1.GenericControlPlaneGroupName` ?

Even in the kubernetes fork, it doesn't seem to be used in such cases. Here is an example of those:

https://github.com/kcp-dev/kubernetes/blob/653a0f16cab7cd12271e7433fc8466686d524f84/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go#L75